documentation/RFC/RFC0002-SubscriptionList-in-Get.md

```diff
@@ -0,0 +1,56 @@
+---
+RFC: RFC0002
+Author: Azure PowerShell team
+Status:  Draft
+SupercededBy: n/a
+Version: 1.0
+---
+
+# List of subscriptions for GET commands
+
+A GET cmdlet is mostly used to obtain an Azure resource or validate its existence in Azure.
+With more and more customers using multiple subscriptions the ability to search for resources accross subscriptions has become a more needed convenience.
+
+We are planning to run GET accross  
```

Comment:
I hope this was intended to be finished with "across all subscriptions within a single Azure Tenant"

As a consultant in many tenants I don't think this functionality should be able to reach across tenants but would be extremely useful within a single tenant

Reply:
Thanks @kilasuit, I missed to finish this sentence.